test/com/latacora/backsaws/ini_test.clj

```diff
@@ -0,0 +1,125 @@
+(ns com.latacora.backsaws.ini-test
+  (:require
+   [com.latacora.backsaws.ini :as i]
+   [clojure.test :as t]))
+
+(def one-kv-outside-section
+  (i/ini-parser "x=1"))
+
+(def empty-section
+  (i/ini-parser "[x]"))
+
+(def empty-section-with-newline
+  (i/ini-parser "[x]\n"))
+
+(def one-section-with-one-kv
+  (i/ini-parser "[xyzzy]\nx = 1"))
+
+(def one-section-with-two-kvs
+  (i/ini-parser "[xyzzy]\nx = 1\ny = 2"))
+
+(def two-sections
+  (i/ini-parser "[xyzzy]\nx = 1\n[iddqd]\ny=2"))
+
+(def sectioned-and-unsectioned-kvs
+  (i/ini-parser "z = 3\n[xyzzy]\nx = 1\n[iddqd]\ny=2"))
```

Comment:
These probably can/should be resources...

Reply:
I dunno, this seems fine to me.